LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
Based on the main source of MIT License (https://opensource.org/licenses/MIT) I think here should be something like this **The MIT License (MIT).**